src/pages/ManageDonationForm/ManageDonationForm.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import { Button, Heading, Flex, Center, useToast } from '@chakra-ui/react';
+import EditItemsList from '../../components/EditItemsList/EditItemsList';
+import ItemInfo from '../../components/ItemInfo/ItemInfo';
+import { PNPBackend } from '../../utils/utils';
+
+const ManageDonationForm = () => {
+  const [items, setItems] = useState([]);
+  const [newEntries, setNewEntries] = useState([]);
+  const toast = useToast();
+
+  useEffect(() => {
+    const renderCards = async () => {
+      const { data } = await PNPBackend.get('/furnitureOptions/');
+      setItems([...data]);
+    };
+    renderCards();
+  }, []);
+
+  const updateItems = () => {
+    const res = newEntries.map(async entry => {
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'res' is assigned a value but never used.